pyproject.toml

```diff
@@ -11,3 +11,25 @@ line_length = 100
 force_sort_within_sections = true
 # Combines "as" imports on the same line
 combine_as_imports = true
+
+[tool.mypy]
+ignore_missing_imports = true
+show_error_codes = true
+disable_error_code = ["attr-defined", "var-annotated"]
```

Comment:
These two error cases produced a lot of type errors, many of which were incorrect because of django typing stubs not working. I've disabled them for now, and if django typing stubs ever get integrated with `django-configurations` properly, we can re-enable them.